### ISSUE TITLE
RTX 3070 stuck at 24 MH/s (non LHR)

### ISSUE THREAD

**Issue by HugoSart** [trexminer/T-Rex#857]
I have two RTX 3070 running t-rex. One of them always runs perfectly fine. The other one, also runs fine, sometimes. Sometimes I check my rig and the hash rate of my card is stuck at 24 MH/s, and the power to 85w. I tried to change the configurations on MSI Afterburner but no matter what I change, nothing changes in the miner.

Before, it was happening just sometimes, and restarting the rig was used to fix the problem. But, now, the card is fully stuck at this lower hash rate, and restarting is not helping.

I'm using Windows 11 and the driver 472.12.

**Comment by trexminer** [comment]
> UPDATE: Manually setting the overclock configurations using --mclock, --cclock and --pl seems to fix the problem. So, this makes me believe that it's actually a t-rex bug.


So, using T-Rex built-in overclock capabilities works whereas using MSI AB causes a problem - why do you think this means it's likely a t-rex bug then?

**Comment by HugoSart** [comment]
Because this happens only with T-Rex and I get full performance on other applications (I also use it for gaming)

**Comment by trexminer** [comment]
Yeah, but gaming and other applications put a very different kind of load on the GPU compared to `ethash`. Have you tried using other miners?

**Comment by charnet3d** [comment]
I had a similar weird behavior happen once in the past, my 3070 ventus 2x (non lhr) always mined with trex until one day when it got stuck building the dag, or it starts mining for a few secs and gets stuck again. No issues gaming.
When I tried other miners I could mine on gminer with no issues, so I left it for a some days like that. It really scared me thinking the memory got corrupt or something.
After some time I tested trex again and it's working like by magic lol.
